usecases/configuration.go

```diff
@@ -178,7 +169,7 @@ func Configure(sys *components.System) ([]json.RawMessage, error) {
 		sys.CoreS = append(sys.CoreS, &newCore)
 	}
 
-	return configurationIn.rawResources, nil
+	return rawResources, nil
 }
 
 // getServicesList() returns the original list of services
```

Comment:
Lines 178-179 can be merged.

Reply:
`MakeServiceMap()` is _never used_ in this library, or in the systems. Remove it.